src/proxy/explain.go

```diff
@@ -71,7 +51,7 @@ func (spanner *Spanner) handleExplain(session *driver.Session, query string, nod
 		return nil, sqldb.NewSQLError(sqldb.ER_SYNTAX_ERROR, "explain only supports SELECT/DELETE/INSERT/UNION")
 	}
 
```

Comment:
Can explain need support `checksum`?  If necessary, modify the doc, else modify the code.